src/plugins/stylesheet/CSSModules.ts

```diff
@@ -32,8 +33,25 @@ export class CSSModulesClass implements Plugin {
 
     public transform(file: File): Promise<any> {
         file.addStringDependency("fuse-box-css");
+
+        if (file.isCSSCached("cssmodules")) {
+            return;
+        }
+        file.bustCSSCache = true;
+
         return new Promise((resolve, reject) => {
             file.loadContents();
+            const context = file.context;
+
+            const paths = [file.info.absDir, ...this.options.paths || []]
+
+            const cssDependencies = context.extractCSSDependencies(file, {
```

Comment:
Don't you think this has to be optional? Some people combine it with SassModule where it has its own dependency tracker.

Reply:
I figured this situation would happen already when using sass but then using postcss for autoprefixing. If both dependency trackers can't be active at the same time, can this be handled internally somehow? I don't think more config is helpful.

Reply:
Can you verify that on your branch?

Reply:
@mdlawson the release for 3.0 is coming tomorrow, let's verify it to avoid bugs. And thank you very much!

Reply:
I'll try check this evening. If this can't make the 3.0 release though that's fine with me.

Reply:
I will try verifying it today too, if I find time, I would like that to be in 3.0 ;-)

Reply:
I've verified the combination of this change and the sass plugin with [this setup](mdlawson/fuse-box-css-modules-example). Changes to both the sass only `_partial.scss` and the css module value included from `colors.scss` both correctly result in hot reloads and expected console output. Is there anything else you'd like me to check out?

I did notice the text logged to the console even without this change is like:
```
  → CSS Dependency: _partial.scss depends on main.scss
```

Which seems the wrong way round to me. `main.scss` depends on `_partial.scss` which is why a change to `_partial.scss` needs to trigger a reload of `main.scss`

Reply:
hm, but you do have main.css -> _partial.css So main.css is a parent file with the dependency `_partial.css`

Reply:
Anyway thank you very much for this PR!